llvm/lib/CodeGen/SelectionDAG/ScheduleDAGSDNodes.cpp

```diff
@@ -888,8 +888,10 @@ EmitSchedule(MachineBasicBlock::iterator &InsertPos) {
     }
 
     if (MI->isCandidateForCallSiteEntry() &&
-        DAG->getTarget().Options.EmitCallSiteInfo)
-      MF.addCallArgsForwardingRegs(MI, DAG->getCallSiteInfo(Node));
+        DAG->getTarget().Options.EmitCallSiteInfo) {
+      // MF.addCallArgsForwardingRegs(MI, DAG->getCallSiteInfo(Node));
+      MF.addCallSiteInfo(MI, DAG->getCallSiteInfo(Node));
+    }
```

Comment:
nit: you don't need the braces. It will be clear if you just remove the commented out line.

Reply:
yup, more generally we should not commit commented out code.

I also agree this deserves to be split as two PRs and merged independently.